models/settings.go

```diff
@@ -51,6 +51,10 @@ type Settings struct {
 
 	// Saas config options
 	SaaS struct {
+		// Platform user email
+		Email string `json:"email"`
+		// Percona platform session Id
```

Comment:
Let's stick to "Percona Platform" in comments/docs, not "Percona platform" or just "Platform" as above

Reply:
Fixed.